### PR TITLE
doc: update env.py api documentation

### PR DESCRIPTION
**Pull request by liuhatry**

### CODE REVIEW

File: bagua/torch_api/env.py [hunk 1]
```diff
@@ -3,62 +3,78 @@
 
 def get_world_size():
     """
-    Returns the number of processes in the current process group
+    Gets the number of processes in the current process group.
```
Comment (NOBLES5E) [inline bagua/torch_api/env.py:6]: Get. Similar modification to other doc strings.

File: bagua/torch_api/env.py [hunk 2]
```diff
@@ -3,62 +3,78 @@
...
+
+    Returns:
+       The ip address of autotune server.
+    """
     return os.environ.get("AUTO_TUNE_SERVER_ADDR")
 
 
 def is_report_metrics_switch_on():
+    """
+    Wheter bagua report switch is on or not.
```
Comment (NOBLES5E) [inline bagua/torch_api/env.py:67]: "Whether"


Also need to explain what "bagua report" is.

File: bagua/torch_api/env.py [hunk 3]
```diff
@@ -3,62 +3,78 @@
...
+    """
     return int(os.environ.get("BAGUA_REPORT_METRICS", 0)) == 1
 
 
 def get_autotune_level():
+    """
+    Get the atuotune level.
```
Comment (NOBLES5E) [inline bagua/torch_api/env.py:74]: Need to explain, or give a ref link to autotune api doc to let the readers know what this is about.

File: bagua/torch_api/env.py [hunk 4]
```diff
@@ -3,62 +3,78 @@
...
+
+    Returns:
+        The autotune level.
+
```
Comment (NOBLES5E) [inline bagua/torch_api/env.py:78]: remove the blank line

File: bagua/torch_api/env.py (hunk 1 above)
Comment (liuhatry) [reply bagua/torch_api/env.py]: done

File: bagua/torch_api/env.py (hunk 2 above)
Comment (liuhatry) [reply bagua/torch_api/env.py]: done

File: bagua/torch_api/env.py [hunk 5]
```diff
@@ -3,62 +3,73 @@
...
 
     Returns:
-        The local size of the node
-
+        The local size of the node.
     """
     return int(os.environ.get("LOCAL_SIZE", 1))
 
 
 def get_autotune_server_addr():
```
Comment (shjwudp) [inline bagua/torch_api/env.py:51]: Inner interface, users DO NOT need to know.

File: bagua/torch_api/env.py [hunk 6]
```diff
@@ -3,62 +3,73 @@
...
+    """
+    Get autotune server addr.
+
+    Returns:
+       The ip address of autotune server.
+    """
     return os.environ.get("AUTO_TUNE_SERVER_ADDR")
 
 
 def is_report_metrics_switch_on():
```
Comment (shjwudp) [inline bagua/torch_api/env.py:61]: Same like get_autotune_server_addr

File: bagua/torch_api/env.py (hunk 3 above)
Comment (shjwudp) [reply bagua/torch_api/env.py:70]: Same like above. I am going to introduce the use of autotune in the tutorial

File: bagua/torch_api/env.py (hunk 5 above)
Comment (liuhatry) [reply bagua/torch_api/env.py:51]: this function has been hidden

File: bagua/torch_api/env.py (hunk 6 above)
Comment (liuhatry) [reply bagua/torch_api/env.py:61]: this function has been hidden

File: bagua/torch_api/env.py (hunk 3 above)
Comment (liuhatry) [reply bagua/torch_api/env.py:70]: this function has been hidden

File: bagua/torch_api/env.py (hunk 5 above)
Comment (liuhatry) [reply bagua/torch_api/env.py:51]: this function has been hidden

**Pull request merged**